djconnectwise/models.py

```diff
@@ -1849,6 +1858,13 @@ def change_sla_state(self, new_stage, calendar, sla):
             self.sla_state.enter(valid_stage, calendar, sla)
 
 
+class NonInternalNoteManager(models.Manager):
+    def get_queryset(self):
+        return super().get_queryset(). \
+            filter(Q(internal_analysis_flag=False) | Q(internal_flag=False)
```

Comment:
This should be either an AND query (both `internal_analysis_flag` is False and `internal_flag` is False), or make it exclude instead of filter.

Reply:
Updated